### DISCUSSION TITLE
Tags cachés & attribution précise de tags chez les joueurs

### DISCUSSION THREAD

**Discussion by Seikame** [fibreville/minotaure_php#75] [Boite à idées]
Ca peut paraître farfelu mais je pensais à d'éventuelles parties d'ATRPG + axées paranoïa et traitrise, genre du setting Loup Garous, ou Among Us. Une façon de rendre ça possible pourrait être de créer des tags dont l'attribution n'apparaîtrait pas sur l'écran d'administration (probablement que le MJ ne saurait même pas à qui ils sont attribués) mais les joueurs sur leur session de personnage seraient bien prévenus de leur attribution de tag.

Une autre prérogative pour ce genre de parties pourrait être l'option de pouvoir contrôler le ratio des tags attribués (par exemple 20% d'imposteurs pour 80% de gens réglos).

**Comment by uurshin** [comment]
Idée géniale ! Ca ouvre des perspectives très intéressantes. Ce qui est bien c'est que depuis que j'ai fait le #53 c'est très facile à rajouter !

Pour le pourcentage d'attribution c'est pas aussi simple par contre donc je séparerai les deux.

**Reply by uurshin** [reply]
D'ailleurs on pourrait permettre l'apparition d'un panneau "caché" dans une autre fenêtre, qui ne serait pas partagé sur le stream.
